tests/wake-format/basic/stdout

```diff
@@ -747,7 +751,12 @@ export def vfoldl (combiningFn: accum => element => accum): accum => Vector elem
 
 
 def loop a s e =
-    if s == e then e else if stopFn (vget a s) then s else loop a (s + 1) e
+    if s == e then
+        e
+    else if stopFn (vget a s) then
+        s
+    else
+        loop a (s + 1) e
```

Comment:
What would the format be if this were `...else if shouldStop then s else t`?  (Basically, does `else if` respect the explosion status of the initial `if`?)

Reply:
yep! `if` explosion propagates to all `if`s in the tree. Previously had to add a case specifically for that. Went ahead and added an explicit test case for it